ansible/all-in-one/roles/xsnippet/tasks/main.yml

```diff
@@ -148,3 +206,56 @@
     enabled: yes
     state: restarted
     scope: user
+
+- name: proxy HTTP and HTTPS from host to containers
+  include_role:
+    name: sockets
+    apply:
+      become: yes
+      become_user: root
+
+- name: systemd unit - restart {{ restart_service_name }} on SSL renewal
+  vars:
+    restart_service_name: container-xsnippet-api.service
```

Comment:
Hmm, we need to restart `container-xsnippet-web.service` though, right? I agree that it's a bit confusing and lacks symmetry, but `container-xsnippet-api.service` is currently where TLS connections are terminated :(

Reply:
Oh yeah, you're right. Nice catch. I'll remove it.